File: DSA/LakshaKundra_590013237/Day_6_Question_2.py
def isStrictlyIncreasing(nums):
    for i in range(1, len(nums)):
        if nums[i] <= nums[i - 1]:
            return False
    return True
print(isStrictlyIncreasing([1, 2, 3, 4]))  
print(isStrictlyIncreasing([1, 2, 2, 3]))   
print(isStrictlyIncreasing([5, 4, 3]))     

File: DSA/VaniJain_590012020/Day6Question2.java
mport java.util.*;
public class IncreasingArray 
{
    public static boolean isStrictlyIncreasing(int[] nums) 
    {
        for (int i=1; i<nums.length; i++) 
        {
            if (nums[i]<=nums[i-1]) 
            {
                return false;
            }
        }
        return true;
    }
    public static void main(String[] args) 
    {
        Scanner scr=new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int n = scr.nextInt();
        int[] nums = new int[n];
        System.out.println("Enter " + n + " integers:");
        for (int i=0; i<n; i++) 
        {
            nums[i]=scr.nextInt();
        }
        boolean x=isStrictlyIncreasing(nums);
        System.out.println("Is the array strictly increasing? " + x);
    }
}
